docs/whatsnew_1.5.8.html

```diff
@@ -171,7 +171,8 @@ <h2>
                <li>Core: Continued function consolidation initiated in 1.5.7.</li>
                <li>Core: Language loading moved into arrays to prevent PHP duplication notices.</li>
                <li>Core: Customer class introduced to consolidate customer table related logic.</li>
-               <li>Core: Plugin manager improvements (<i>more details here</i>).</li>
+               <li>Core: Plugin manager improvements (<i>FIXME more details here</i>).</li>
+               <li>Core: Laravel inclusion (<i>FIXME more details here</i>).</li>
```

Comment:
There's no end-user benefit to mentioning Laravel yet.

Reply:
It was just a placeholder for Ian.  I'll let him handle this - removed.